shared_model/builders/protobuf/queries.hpp

```diff
@@ -116,28 +116,38 @@ namespace shared_model {
       }
 
       auto getAccountTransactions(
-          const interface::types::AccountIdType &account_id) const {
+          const interface::types::AccountIdType &account_id,
+          const interface::Transaction::HashType &pager_tx_hash =
```

Comment:
Why it passes as a param and doesn't calculated?

Reply:
pager.tx_hash can be empty as specification.
When we want to get transaction from the top, we cannot specify any tx_hash.

Reply:
But new model hash type is not stricted for its size. I uses this feature but maybe it can lead to some misunderstandings for developers because libs hast_t has strict size.

Reply:
Do we have some kind of spec for pager?

Reply:
Here is spec: https://hackmd.io/KwBgzAJhCMIIYFoBM0wHYEBYBmAOECcc2BAxqUrgGya5hyZpJA==?both